### PR TITLE
fix(Slider): added event param to onChange

### LINKED ISSUE #8963: Slider: Add event as the first parameter of  `onChange`

### PR DESCRIPTION
**Pull request by thatblindgeye**
**What**: Closes #8963 

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -55,6 +55,12 @@ export interface SliderProps extends Omit<React.HTMLProps<HTMLDivElement>, 'onCh
   min?: number;
   /** Value change callback. This is called when the slider value changes. */
   onChange?: (
+    event:
```
[the remark this answers is not in the document]
Comment (thatblindgeye) [reply packages/react-core/src/components/Slider/Slider.tsx]: Ah good call. Added a type for the event param in the latest push

**Review by nicolethoen — COMMENTED** [review]
I think we will need to define the `SliderOnChangeEvent` in the docs. If we cannot add it to the `propComponents` array, then adding it directly to the markdown as a code snippet or something will suffice for the short term.

**Comment by thatblindgeye** [comment]
@nicolethoen couldn't add it to the propComponents currently, so put a section after "Examples" to describe it

**Review by kmcfaul — APPROVED** [review]

File: packages/react-core/src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -19,6 +19,13 @@ export interface SliderStepObject {
   value: number;
 }
 
+export type SliderOnChangeEvent =
```
Comment (tlabaj) [inline packages/react-core/src/components/Slider/Slider.tsx:22]: I think this is ok... I don't know that is necessary. Would we want to upstate other event type definitions elsewhere to be the same for consistency?   What ever we decide, I would like to revisit the slider docs once this org [issue](https://github.com/patternfly/patternfly-org/issues/3423) gets resolve.  I don't love the hand generated  "Types" section in the docs.

**Pull request merged**